lib/plugins/exchange/ccxt.js

```diff
@@ -49,4 +54,36 @@ function calculatePrice (side, amount, orderBook) {
   throw new Error('Insufficient market depth')
 }
 
-module.exports = { trade }
+function _getMarkets (exchangeName, availableCryptos) {
+  try {
+    const exchange = new ccxt[exchangeName]()
+    const cryptosToQuoteAgainst = ['USDT']

```

Comment:
Is there a reason we're quoting against USDT instead of USD? What happens if it depegs?

Reply:
We're quoting against both USDT and USD. This variable represents a list of cryptos that operators want to have available to trade with on the exchange (e.g. USDT/BTC pair). USDT comes as a specific request in the comments of the issue card